src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/EnergyDistribution/KalbachMann/Data/Builder/src/interpolationParameter.hpp

```diff
@@ -0,0 +1,10 @@
+Builder& interpolationParameter( int interpolationParameter ){
+  if( interpolationParameter <= 0 ){
```

Comment:
allowed values are 1 or 2, with values of higher than 10 folding ND in it. Maybe just allow INTT equal to 1 or 2 and add numberDiscretePhotons, and calculate the folded value in ACEify

Reply:
Done